src/ripple/app/tx/impl/Transactor.cpp

```diff
@@ -567,8 +568,9 @@ void removeUnfundedOffers (ApplyView& view, std::vector<uint256> const& offers,
     }
 }
 
+/** Reset the context, discarding any changes made and adjust the fee */
 void
-Transactor::claimFee (XRPAmount& fee, TER terResult, std::vector<uint256> const& removedOffers)
+Transactor::reset(XRPAmount &fee)
```

Comment:
Consider returning `fee` rather than using an in/out parameter. Given the purpose of this function and the function name, it's not obvious at the call site that `fee` is adjusted and it helps us reason about invariants to know that it can be.

Also Nit: formatting, should be `XRPAmount& fee`

Reply:
I contemplated returning the updated fee; in this limited use case, it would almost certainly be fine, but I'd feel *much* better doing it if we were on C++ and I could tag the function as `[[nodiscard]]`.

I'll go with your suggestion.